test_assets/Asset_1.json

```diff
@@ -1,15 +1,15 @@
 {
     "id": "Asset_1",
-    "name": "Iron_(PUBCHEM)_treats_Aceruloplasminemia",
-    "description": "Iron_(PUBCHEM)_treats_Aceruloplasminemia",
+    "name": "NeverShow: Iron_(PUBCHEM)_treats_Aceruloplasminemia",
+    "description": "NeverShow: Iron_(PUBCHEM)_treats_Aceruloplasminemia",
     "tags": [],
     "input_id": "MONDO:0011426",
-    "input_name": "Aceruloplasminemia",
-    "input_category": null,
-    "predicate_id": "biolink:treats",
+    "input_name": "MONDO:0011426",
+    "input_category": "biolink:Disease",
+    "predicate_id": "treats",
```

Comment:
Should this include the `biolink:` prefix?

Reply:
in the last review, you asked me to remove the biolink prefix :) -- for input_id, output_id, predicate, qualified_predicate -should we have the biolink prefix or not?

Reply:
The issue before was that some predicate_ids had this: `biolink:biolink:<predicate>`, and then the predicate name also had `biolink:<predicate>`. Now neither predicate_id nor predicate_name have the biolink prefix.

Reply:
For your question about the other fields, yes, I think those should have the biolink prefix.